tests/sentry/integrations/slack/test_link_team.py

```diff
@@ -109,6 +135,9 @@ def setUp(self):
             channel_name=self.channel_name,
             response_url=self.response_url,
         )
+        self.team = self.create_team(
+            organization=self.organization, name="Mariachi Band", members=[self.user]
+        )
```

Comment:
Create a team in `setUp` so we have a team show in the dropdown and don't error with `404`.